sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -146,7 +148,10 @@ export class AttestationToken {
     get certificateSha256Thumbprint(): string | undefined;
     get certificateThumbprint(): string | undefined;
     get contentType(): string | undefined;
-    static create(body: string, signer?: AttestationSigningKey): AttestationToken;
+    static create(params: {
+        body?: string;
+        signer?: AttestationSigningKey;
+    }): AttestationToken;
     get critical(): boolean | undefined;
     get expirationTime(): Date | undefined;
     get_body(): any;
```

Comment:
can we rename this to `getBody` if it is not coming from the generated layer.

Reply:
*doh* - cut&paste from python.